app/graphql/mutations/invites/revoke.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Mutations
+  module Invites
+    class Revoke < BaseMutation
+      include AuthenticableApiUser
+      include RequiredOrganization
+
+      graphql_name 'RevokeInvite'
+      description 'Revokes a invite'
+
+      argument :id, ID, required: true
+
+      type Types::Invites::Object
+
+      def resolve(id:)
+        result = ::Invites::RevokeService.new(context[:current_user]).call(id)
```

Comment:
you do not pass the `organization` here?

Reply:
Nope, I don't need the orga in the revoke service. Why ?

Reply:
when you search for the invite you should scope it to the `current_organization` for security reasons!